### PR TITLE
Payment UG updates 

### PR DESCRIPTION
**Pull request by KinTatHo**
Fixed issues #156 , #140  , #131, #161

### CODE REVIEW

**Review by jessicawyz — CHANGES_REQUESTED** [review]

File: docs/UserGuide.md [hunk 1]
```diff
@@ -156,23 +159,59 @@ Format: `view -id {ID}`
...
-This will reset a student's payment to 0 by searching for their `id`. 
-Format: `resetpayments -id {ID}`
-Examples:
-* `resetpayments -id 88888` would reset the payment for the student whose id is 88888 if it exists.
+### Adding Payments: `addpayment`
+
+Record new payments easily with the `addpayment` command, keeping track of what each student owes. This command simplifies the maintenance of financial records, ensuring you never overlook an outstanding payment.
+
+**Format:** `addpayment -id {ID} -payment {AMOUNT}`
+**Example:** `addpayment -id 000001 -payment 150` or `addpayment -id 1 -payment 150` would add a payment of 100 to the student whose ID is '#000001' if it exists. <br>
```
[the remark this answers is not in the document]
Comment (KinTatHo) [reply docs/UserGuide.md:171]: oh right, will fix it

File: docs/UserGuide.md [hunk 2]
```diff
@@ -156,23 +159,59 @@ Format: `view -id {ID}`
...
+
+<box type="info" seamless>
+* If the amount marked as paid exceeds the total outstanding payment, the total outstanding payment will be set to 0. <br>
+* If the amount owed by student is already 0, you are still allowed to enter the command, but there will be no changes. <br>
+* All payments added will be rounded off to the nearest $0.01.
+</box>
+
+### Resetting Payments: `resetpayments`
+
+Use the `resetpayments` command to clear a student's outstanding payment amount, useful for starting anew or correcting mistakes. This command ensures your financial records remain accurate and up-to-date.
```
[the remark this answers is not in the document]
Comment (KinTatHo) [reply docs/UserGuide.md:201]: okay makes sense, will change

**Review by MarcusGitty — APPROVED** [review]
LGTM

**Review by jessicawyz — APPROVED** [review]
LGTM!

**Pull request merged**